constructor/osxpkg.py

```diff
@@ -230,6 +233,8 @@ def create(info, verbose=False):
             ndist = dist.name
         fresh_dir(PACKAGE_ROOT)
         cph_e(join(CACHE_DIR, fn), join(pkgs_dir, dname))
+        if keep_pkgs:
```

Comment:
The `.pkg` installer copies in just the extracted packages. But if `keep_pkgs` is True we want the tarballs as well.